features/vendor_views_closed_auction.feature

```diff
@@ -4,29 +4,25 @@ Feature: Vendor views closed auctions
   So that I see who won
 
   Scenario: I am the winner
-    Given there is a closed auction
-    And I am an authenticated vendor
-    And I have placed the lowest bid
+    Given I am an authenticated vendor
+    And I have won an auction
     When I visit the auction page
     Then I should see the auction had a winning bid with name
     And I should see I am the winner
     And I should see when the auction started
     And I should see when the auction ended
     And there should be meta tags for the closed auction
-    And an email notification is sent to the winning bidder
```

Comment:
I think that happens because delayed jobs run immediately in the test env. so in order to not have that happen, we'd set them up to *not* run immediately in a test